gnome-initial-setup/pages/password/gis-password-page.c

```diff
@@ -206,6 +206,26 @@ validate (GisPasswordPage *page)
   return FALSE;
 }
 
+static void
+page_visibility_changed (GtkWidget       *w,
+                         GParamSpec      *pspec,
+                         GisPasswordPage *page)
```

Comment:
You’re not actually passing the page as user_data to the signal callback, so this is technically not correct.
Also you declare a variable with the same name in th inner block.